dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2094 @@
+
+        Group group = workflowService.createWorkflowRoleGroup(context, collection, "reviewer");
+
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/workflowGroups/reviewer"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/workflowGroups/reviewer"))
+                        .andExpect(status().isNoContent());
+    }
+
+    // This is currently not supported in DSpace API
+    @Ignore
+    @Test
+    public void deleteCollectionWorkflowGroupTestCollectionAdmin() throws Exception {
+
+        Group group = workflowService.createWorkflowRoleGroup(context, collection, "reviewer");
+
+        authorizeService.addPolicy(context, collection, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/workflowGroups/reviewer"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/workflowGroups/reviewer"))
+                        .andExpect(status().isNoContent());
+    }
+
+    @Test
+    public void deleteCollectionWorkflowGroupUnAuthorizedTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group group = workflowService.createWorkflowRoleGroup(context, collection, "reviewer");
+        context.restoreAuthSystemState();
+
+        getClient().perform(delete("/api/core/collections/" + collection.getID() + "/workflowGroups/reviewer"))
+                   .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void deleteCollectionWorkflowGroupForbiddenTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group group = workflowService.createWorkflowRoleGroup(context, collection, "reviewer");
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/workflowGroups/reviewer"))
+                        .andExpect(status().isForbidden());
+    }
+
+
+    @Test
+    public void deleteCollectionWorkflowGroupNotFoundTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group group = workflowService.createWorkflowRoleGroup(context, collection, "reviewer");
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + UUID.randomUUID() + "/workflowGroups/reviewer"))
+                        .andExpect(status().isNotFound());
+    }
+
```

Comment:
due to the custom permission mechanism that exists for these communities / collections group we need to have IT also veryfing the ability to update such groups and add/remove member to them. If this is to much work for this specific PR I'm fine with postpone that to a dedicated ticket (please create it if so)

Reply:
@abollini Are you indicating that we should create a test that community / collection admins can add members to these groups that are linked to a community / collection ?

Reply:
yes, your understanding is correct. Please include adding and removing of members as well as edit the group metadata. Again I'm also fine with a separate ticket for that

Reply:
yes, your understanding is correct. Please include adding and removing of members as well as edit the group metadata. Again I'm also fine with a separate ticket for that